src/ol/graticule.js

```diff
@@ -134,6 +161,41 @@ ol.Graticule = function(opt_options) {
    */
   this.projectionCenterLonLat_ = null;
 
+  /**
+   * @type {boolean}
+   * @private
+   */
+  this.showLabels_ = options.showLabels !== undefined ?
+      options.showLabels : false;
+
+  /**
+   * @type {?Function}
+   * @private
+   */
+  this.lonLabelFormatter_ = options.lonLabelFormatter !== undefined ?
+      options.lonLabelFormatter : null;
+
+  /**
+   * @type {number} between 0.0 and 1.0
+   * @private
+   */
+  this.lonLabelPosition_ = options.lonLabelPosition !== undefined ?
+      ol.math.clamp(options.lonLabelPosition, 0.0, 1.0) : 1.0;
+
+  /**
+   * @type {?Function}
+   * @private
+   */
+  this.latLabelFormatter_ = options.latLabelFormatter !== undefined ?
+      options.latLabelFormatter : null;
+
+  /**
+   * @type {number} between 0.0 and 1.0
+   * @private
+   */
+  this.latLabelPosition_ = options.latLabelPosition !== undefined ?
+      ol.math.clamp(options.latLabelPosition, 0.0, 1.0) : 1.0;
```

Comment:
Does it actually make sense to have default label positions `1.0` ?
This value makes the labels float outside(/on the edge of) the viewport, so anyone who wants to use this would have to specify more sensible value anyway..